docs/docs/adding-a-shopping-cart-with-snipcart.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Adding a Shopping Cart with Snipcart
+---
+
+Snipcart is a shopping cart solution designed to be dropped neatly into any web project. Including it in your project allows any HTML you write to instantly become a "buy button" with the addition of several Snipcart-specific attributes.
+
+Combine it with a source of products (like a CMS or an e-commerce platform like [Etsy](https://www.etsy.com/)) and a payment processer (like [Stripe](https://www.stripe.com/)) to build a complete e-commerce experience for your customers.
+
+## Prerequisites
+
+To get started, you'll need
+
+- a Gatsby site with [`gatsby-plugin-snipcart`](/packages/gatsby-plugin-snipcart/) installed
+- a [Snipcart](https://snipcart.com/) account
+- your Snipcart test API key
+- products to sell
+
+Installing the plugin adds Snipcart's shopping cart JavaScript for you, so you can get right to building your e-commerce site. It's okay if you're not sure what you'd like to sell quite yet. Using sample products is fine to begin with!
+
+## Defining Products
+
+Adding products with Snipcart involves writing HTML representing your product and adding a set of attributes to that HTML. You might write something similar to the following code block for each item in your catalogue.
```

Comment:
Switched to JSX! I didn't add a specific file because this would depend on where you listed your products. Maybe something a little less prescriptive like "wherever you list items for sale..."